app/web/markdown/blog/2025/07/01/releasing-couchers-v1.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "A new chapter: Couchers is officially out of Beta!"
+slug: releasing-couchers-v1
+description: "We are out of Beta and in v1: we're releasing a new strategy around safe & active community instead of bashing our competitors, a fancy redesigned landing page, and a bunch of new features to make core couch surfing functionality better!"
+date: 2025/07/01
+author: Aapeli
+---
+
+*Quick summary: we are out of Beta and in v1, we're releasing a new strategy around safe & active community instead of bashing our competitors, a fancy redesigned landing page, and a bunch of new features to make core couch surfing functionality better! Share the platform with your friends and let's grow the community together!*
+
```

Comment:
```suggestion
*Quick summary: we are out of Beta and into version 1, we're releasing a new strategy around safe & active community instead of bashing our competitors, a fancy redesigned landing page, and a bunch of new features to make core couch surfing functionality better! Share the platform with your friends and let's grow the community together!*

```